README.md

```diff
@@ -0,0 +1,111 @@
+| `roundcube_des_key`              | :heavy_check_mark: |  | This key is used to encrypt the users imap password which is stored in the session record (and the client cookie if remember password is enabled). Please provide a string of exactly 24 chars.                                                                                                                                                                                |
+| `roundcube_install_version`          | :heavy_multiplication_x:       |       `1.3.3`      | Version to install                                                                                                                                                                             |
+| `roundcube_user`              | :heavy_multiplication_x: | `www-data` | Name of the user to be used for roundcube                                                                                                                                                                |
+| `roundcube_group`                 | :heavy_multiplication_x: | `www-data` | Group to be used for roundcube                                                                                                                                                           |
+| `roundcube_validate_certs`           | :heavy_multiplication_x: | `true`      | Should roundcube validate certs during connection to the mail server |
+| `roundcube_sql_host`                | :heavy_multiplication_x: |        `localhost`      | Host of for the database                                                                                                                                                                |
+| `roundcube_sql_user`             | :heavy_multiplication_x: |        `roundcube`      | Database user                                                                                                                                                                     |
+| `roundcube_sql_database`               | :heavy_multiplication_x: |              | Database name                                                                                                                                                               |
+| `roundcube_default_host`  | :heavy_check_mark:       |     `ssl://{{ mailserver_fqdn_hostname | default('localhost') }}`         | Imap server                                                                                                                                                            |
+| `roundcube_default_port`     | :heavy_check_mark:       |       `993`       | Imap port                                                                                                                                                            |
```

Comment:
maybe rename this
from the name I would think this is the port where I can reach roundcube
same for host

Reply:
fixed